package.json

```diff
@@ -8,80 +8,86 @@
-    "@angular/core": "^11.2.9",
-    "@angular/forms": "^11.2.9",
-    "@angular/platform-browser": "^11.2.9",
-    "@angular/platform-browser-dynamic": "^11.2.9",
-    "@angular/router": "^11.2.9",
+    "@angular/animations": "^14.2.5",
+    "@angular/common": "^14.2.5",
+    "@angular/compiler": "^14.2.5",
+    "@angular/core": "^14.2.5",
+    "@angular/forms": "^14.2.5",
+    "@angular/platform-browser": "^14.2.5",
+    "@angular/platform-browser-dynamic": "^14.2.5",
+    "@angular/router": "^14.2.5",
     "@biesbjerg/ngx-translate-extract": "^7.0.3",
     "@biesbjerg/ngx-translate-extract-marker": "^1.0.0",
     "@ngx-formly/bootstrap": "^5.12.4",
     "@ngx-formly/core": "^5.12.4",
     "@ngx-translate/core": "^13.0.0",
     "@ngx-translate/http-loader": "^6.0.0",
-    "bootstrap": "^4.6.0",
+    "@types/marked": "^4.0.7",
+    "bootstrap": "^4.6.2",
     "crypto-js": "^3.3.0",
     "dot-prop": "^6.0.1",
+    "easymde": "^2.18.0",
     "elliptic": ">=6.5.4",
     "font-awesome": "^4.7.0",
     "handlebars": "^4.7.7",
     "hosted-git-info": ">=2.8.9",
     "http-proxy": ">=1.18.1",
     "kind-of": "^6.0",
     "lodash": "^4.17.21",
-    "lodash-es": "4.17.14",
+    "lodash-es": "^4.17.21",
+    "marked": "^4.1.1",
     "minimist": ">=0.2.1",
     "moment": "^2.29.1",
-    "ngx-bootstrap": "^6.2.0",
-    "ngx-spinner": "^10.0.1",
-    "ngx-toastr": "^13.2.1",
+    "ngx-bootstrap": "^9.0.0",
+    "ngx-spinner": "^14.0.0",
+    "ngx-toastr": "^15.2.0",
     "npm-registry-fetch": "^9.0.0",
     "path-parse": ">=1.0.7",
     "postcss": ">=8.2.10",
     "rxjs": "^6.6.7",
     "serialize-javascript": "^5.0.1",
-    "simplemde": "^1.11.2",
     "socket.io": "^3.1.2",
     "ssri": ">=8.0.1",
     "tree-kill": ">=1.2.2",
-    "tslib": "^2.2.0",
+    "tslib": "^2.4.0",
     "url-parse": "^1.5.3",
     "y18n": ">=4.0.1",
-    "zone.js": "~0.10.2"
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.1102.9",
-    "@angular-devkit/build-ng-packagr": "^0.1002.0",
-    "@angular/cli": "^11.2.8",
-    "@angular/compiler-cli": "^11.2.9",
-    "@angular/language-service": "^11.2.9",
+    "@angular-devkit/build-angular": "^14.2.5",
+    "@angular-eslint/builder": "14.1.2",
+    "@angular-eslint/eslint-plugin": "14.1.2",
+    "@angular-eslint/eslint-plugin-template": "14.1.2",
+    "@angular-eslint/schematics": "14.1.2",
+    "@angular-eslint/template-parser": "14.1.2",
+    "@angular/cli": "^14.2.5",
+    "@angular/compiler-cli": "^14.2.5",
+    "@angular/language-service": "^14.2.5",
     "@ngx-formly/schematics": "^5.12.4",
     "@types/jasmine": "^3.6.9",
-    "@types/marked": "^2.0.3",
     "@types/node": "^12.20.7",
-    "@types/simplemde": "^1.11.7",
-    "codelyzer": "^6.0.0",
+    "@typescript-eslint/eslint-plugin": "5.37.0",
+    "@typescript-eslint/parser": "5.37.0",
+    "eslint": "^8.23.1",
     "husky": "^4.3.8",
     "jasmine-core": "~3.6.0",
     "jasmine-spec-reporter": "~5.0.0",
```

Comment:
`"jasmine-spec-reporter": "~7.0.0"`

Reply:
Unfortunately not we need to fix some deprecations in the test files first.